tests/test_intercept.js

```diff
@@ -106,6 +106,28 @@ test("when request's content-type is json: reply callback's requestBody should a
   scope.done()
 })
 
+test("adds a leading slash when the path doesn't include one", function(t) {
+  t.plan(1)
+
+  const scope = nock('http://example.test')
+    .get('no-leading-slash')
```

Comment:
JavaScript is my first language these days, but I'm a polyglot and there are some really wonderful gems in The Zen of Python which I think the JS world could really benefit from.

> Errors should never pass silently.
> There should be one-- and preferably only one --obvious way to do it.

(`python -c 'import this'`)

Reply:
^^ I had the same thoughts but in the end decided to accept the change because Kevin put so much effort into it. But throwing a helpful error message instead might be a good compromise?

Reply:
I appreciate the feeling @gr2m, but I think @paulmelnikow may be right. Raising an error will inform the user that there's a **right** way to do things in nock and will help educate them on how to use it. After all, we don't want them to think nock will handle every single deviation from the norm they may have.

I'll do the change if you both agree.

Reply:
yes please 👍 thanks!!